shaders/frag/edgeFracking.glsl

```diff
@@ -0,0 +1,72 @@
+#version 330
+
+uniform sampler2D tex0;
+uniform sampler2D lastFrame;
+uniform float windowHeight;
+uniform float windowWidth;
+uniform float time;
+in vec2 fragTexCoord;
+out vec4 outputColor;
+
+
+uniform float sobelForce = 0.012;
+uniform float newFrameStrength = 0.1;
+uniform int millisBetween = 4;
+
+mat3 x_kernel;
+mat3 y_kernel;
+float x_step = 1/windowWidth;
+float y_step = 1/windowHeight;
+
+void main() {
+  x_kernel[0] = vec3(-1, 0, 1);
+  x_kernel[1] = vec3(-2, 0, 2);
+  x_kernel[2] = vec3(-1, 0, 1);
+
+  y_kernel[0] = vec3( 1, 2, 1);
+  y_kernel[1] = vec3( 0, 0, 0);
+  y_kernel[2] = vec3(-1,-2,-1);
+
+  vec4 x_aggregator = vec4(0);
+  x_aggregator += x_kernel[0][0] * texture(lastFrame, vec2((x_step * -1 + fragTexCoord.x),(y_step*-1 + fragTexCoord.y)));
```

Comment:
We should DRY this up with a `kernel` function, and maybe `dot` if it works on this?